AK/StringUtils.h

```diff
@@ -38,6 +39,13 @@ enum class TrimWhitespace {
     No,
 };
 
+enum class OnSplit : unsigned {
```

Comment:
OnSplit sounds like a callback (`on_foo = ...` is very common across the codebase). Something like `SplitBehavior` seems better.